<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddLightCone</class>
 <widget class="QWidget" name="AddLightCone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AddLightCone</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/icon.png</normaloff>:/res/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>25</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Concept</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="concept">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>25</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>50</number>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>25</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Rairty</string>
   </property>
  </widget>
  <widget class="QComboBox" name="rairty">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>25</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>5</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/rairty/res/rairty/5star.png</normaloff>:/rairty/res/rairty/5star.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/rairty/res/rairty/4star.png</normaloff>:/rairty/res/rairty/4star.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/rairty/res/rairty/3star.png</normaloff>:/rairty/res/rairty/3star.png</iconset>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>75</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Path</string>
   </property>
  </widget>
  <widget class="QComboBox" name="path">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>75</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <item>
    <property name="text">
     <string>Abundance</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/path/res/path/abundance.png</normaloff>:/path/res/path/abundance.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Destruction</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/path/res/path/destruction.png</normaloff>:/path/res/path/destruction.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Erudition</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/path/res/path/erudition.png</normaloff>:/path/res/path/erudition.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Harmony</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/path/res/path/harmony.png</normaloff>:/path/res/path/harmony.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nihility</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/path/res/path/nihility.png</normaloff>:/path/res/path/nihility.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Preservation</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/path/res/path/preservation.png</normaloff>:/path/res/path/preservation.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>The Hunt</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/path/res/path/thehunt.png</normaloff>:/path/res/path/thehunt.png</iconset>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>125</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Level</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="level">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>125</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>80</number>
   </property>
   <property name="value">
    <number>80</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>75</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Superimpose Level</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="superimposelevel">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>75</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>125</y>
     <width>65</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Base HP</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="BaseHP">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>125</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <double>1500.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>175</y>
     <width>65</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Base ATK</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="BaseATK">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>175</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <double>1500.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>175</y>
     <width>65</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Base DEF</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="BaseDEF">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>175</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <double>1500.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="CritDMG">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>345</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="maximum">
    <double>2.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.001000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="EffectHitRate">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>295</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.001000000000000</double>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>HP%</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>EffectRES</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>345</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>CritDMG</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ATKBoostPercentage">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>245</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.001000000000000</double>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>295</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>EffectHitRate</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>245</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ATK%</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="CritRate">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.001000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="DEFBoostPercentage">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>245</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.001000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="EffectRES">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.001000000000000</double>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>295</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>BreakEffect</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>245</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>DEF%</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>CritRate</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="BreakEffect">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>295</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="maximum">
    <double>2.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.001000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="HPBoostPercentage">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>270</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.001000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SkillBoost (No more than two)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Skill</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="skill">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>340</width>
     <height>80</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter LightCone skill</string>
   </property>
  </widget>
  <widget class="QPushButton" name="configuebtn">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>500</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font: 16pt &quot;Meiryo UI&quot;;
background-color: rgb(231, 231, 231);
border-radius: 15px;
border:1px solid rgb(0, 0, 0)
}
QPushButton:hover
{
    padding-left:3px;
    padding-top:3px;
}
QPushButton:pressed
{
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string>Configue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calcelbtn">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>500</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font: 20pt &quot;Meiryo UI&quot;;
background-color: rgb(231, 231, 231);
border-radius: 15px;
border:1px solid rgb(0, 0, 0)
}
QPushButton:hover
{
    padding-left:3px;
    padding-top:3px;
}
QPushButton:pressed
{
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
